app/models/manageiq/providers/autosde/storage_manager/host_initiator_group.rb

```diff
@@ -6,7 +6,17 @@ def self.raw_create_host_initiator_group(ext_management_system, options = {})
       :name           => options['name'],
       :storage_system => PhysicalStorage.find(options['physical_storage_id']).ems_ref
     )
-    ext_management_system.autosde_client.HostClusterApi.host_clusters_post(host_initiator_group_to_create)
-    EmsRefresh.queue_refresh(ext_management_system)
+    task_id =
+      ext_management_system.autosde_client.HostClusterApi.host_clusters_post(host_initiator_group_to_create).task_id
+
+    options = {
+      :target_class   => nil,
+      :target_id      => nil,
+      :ems_id         => ext_management_system.id,
+      :native_task_id => task_id,
+      :interval       => 10.seconds,
```

Comment:
Any reason this is 10sec where the rest are 20sec?

Reply:
all the post are 10 and delete 20